app/ts/library/ethereum.ts

```diff
@@ -0,0 +1,22 @@
+import { BigNumber, ethers } from 'ethers'
+import { TransactionReceipt, TransactionResponse, TransferTransactionResponse } from '../types.js'
+import { ERC20ABI } from './ERC20ABI.js'
+
+export function isTransferTransaction(txResponse: TransactionResponse): txResponse is TransferTransactionResponse {
+	return txResponse.data.startsWith('0xa9059cbb')
```

Comment:
Just in case, it would be best to do a case insensitive match here.
```suggestion
	return txResponse.data.toLower().startsWith('0xa9059cbb')
```